pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1085,8 +1085,8 @@ func getVolumeID(pv *v1.PersistentVolume, volumeName string) (string, error) {
 
 	// Get volID from pvspec if available, else fill it from volumename.
 	if pv != nil {
-		if pv.Annotations["VolID"] != "" {
-			volumeID = pv.Annotations["VolID"]
+		if pv.Annotations[heketiVolIDAnn] != "" {
```

Comment:
So, I would assume this never worked or was the annotation changed some point in time?

Reply:
@gnufied annotation changed just before the last patch which I corrected now.

Reply:
Can you link the patch where annotation was changed? I am wondering about backward compatibility.

Reply:
https://github.com/kubernetes/kubernetes/pull/56315 this patch brought that annotation, till then I was using `volid` as the annotation. Reg#backward compatibility I have tested and verified it works. @gnufied thanks!